drivers/mcp2515/include/mcp2515_params.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mcp2515
+ * @{
+ *
+ * @file
+ * @brief       Parameters for the CAN driver implementation
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @}
+ */
+
+#ifndef MCP2515_PARAMS_H
+#define MCP2515_PARAMS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "can/device.h"
+#include "candev_mcp2515.h"
+
+#include "board.h"
+
+/**
+ * @name    Set default configuration parameters for the MCP2515
+ * @{
+ */
+#ifndef MCP2515_PARAM_SPI
+#define MCP2515_PARAM_SPI SPI_DEV(0)
+#endif
+
+#ifndef MCP2515_PARAM_SPI_MODE
+#define MCP2515_PARAM_SPI_MODE SPI_MODE_0
+#endif
+
+#ifndef MCP2515_PARAM_SPI_CLK
+#define MCP2515_PARAM_SPI_CLK SPI_CLK_10MHZ
+#endif
+
+#ifndef MCP2515_PARAM_CS
+#define MCP2515_PARAM_CS SPI_HWCS(0)
```

Comment:
No, afaik there is no hardware CS on my AVR, and I agree with you that it would be better to set a software CS as a default here. 
My guess is that the STM32 that this driver was originally developed for had a hardware CS.